shoop/admin/modules/products/views/forms.py

```diff
@@ -134,3 +142,115 @@ def save(self):
                     # TODO: Should we maybe _not_ drop the assignment?
                     pass
         self.product.clear_attribute_cache()
+
+
+class ProductMediaForm(MultiLanguageModelForm):
+    class Meta:
+        model = ProductMedia
+        fields = (
+            "file",
+            "ordering",
+            "external_url",
+            "public",
+            "title",
+            "description",
+            "purchased",
+            "shops",
+            "kind"
+        )
+
+    def __init__(self, **kwargs):
```

Comment:
Add a docstring describing `allow_only`? It's not very intuitive unless you know.